### ISSUE TITLE
original xbox compatibility

### ISSUE THREAD

**Issue by GreenHawk2021** [darthcloud/BlueRetro#120]
Hello! Thanks for your project! Will there be any updates to use Blueretro with Original Xbox? Thanks!

**Comment by darthcloud** [comment]
No, see answer here https://github.com/darthcloud/BlueRetro/discussions/84
